sdk/core/core-util/CHANGELOG.md

```diff
@@ -2,13 +2,9 @@
 
 ## 1.6.1 (Unreleased)
 
-### Features Added
-
-### Breaking Changes
-
 ### Bugs Fixed
 
-### Other Changes
+- Fixed Base64 decode in `stringToUint8Array` in browser so that it supports binary that contains UTF-16 characters which have a continuation token.
```

Comment:
![bitmoji](https://sdk.bitmoji.com/render/panel/ca01c138-22c5-46df-9e1f-e738d9bcf2d1-7da08bc9-c7bd-48f0-a0f4-05d7c232ad5b-v1.png?transparent=1&palette=1&width=150)

Reply:
I cannot add bitmoji, so here's the best I can do :shipit: